doc/source/rllib/package_ref/replay-buffers.rst

```diff
@@ -0,0 +1,57 @@
+.. _replay-buffer-reference-docs:
+
+ReplayBuffer API
+================
+
+These classes don't take into account the separation of experiences from different policies, which we use in multi-agent settings.
```

Comment:
"The following classes ....."

".. Multi-agent replay buffers will be explained further below."